core/unit_test/TestComplex.hpp

```diff
@@ -688,6 +687,24 @@ TEST(TEST_CATEGORY, complex_structured_bindings) {
   test.testit();
 }
 
+constexpr bool basics_are_constexpr() {
+  [[maybe_unused]] Kokkos::complex<double> from_single{1.2};
+  [[maybe_unused]] Kokkos::complex<double> from_both{1.2, 3.4};
+  [[maybe_unused]] Kokkos::complex<double> from_none{};
+
+  [[maybe_unused]] auto from_copy_assign = from_both;
+  [[maybe_unused]] auto from_copy_constr(from_both);
+
+  [[maybe_unused]] auto from_move_assign = std::move(from_both);
+  [[maybe_unused]] auto from_move_constr(std::move(from_single));
+
+  [[maybe_unused]] Kokkos::complex<double> from_real = 4.;
```

Comment:
I would want to see some static assertions about the real and imaginary parts in that test

Reply:
Done, the test is now rock solid.